modules/gapi/src/backends/fluid/gfluidcore_simd_sse42.hpp

```diff
@@ -0,0 +1,753 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#if !defined(GAPI_STANDALONE)
```

Comment:
Yes, I agree. However it's not necessary to add one more guard, because whole `gfluidcore.cpp` file already is under this guard. Please take a look [here](https://github.com/opencv/opencv/blob/ceb94d52a104c0c1287a43dfa6ba72705fb78ac1/modules/gapi/src/backends/fluid/gfluidcore.cpp#L7)